var/spack/repos/builtin/packages/llvm/package.py

```diff
@@ -560,6 +562,8 @@ class Llvm(CMakePackage, CudaPackage):
     patch("add-include-for-libelf-llvm-12-14.patch", when="@12:14")
     patch("add-include-for-libelf-llvm-15.patch", when="@15")
 
+    patch("sanitizer-platform-limits-posix-xdr-macos.patch", when="@14 %apple-clang@15")
```

Comment:
Probably needed for newer versions too
```suggestion
    patch("sanitizer-platform-limits-posix-xdr-macos.patch", when="@14 %apple-clang@15:")
```
Can you check if all versions of LLVM 14 work and none of 13 or 15 work?

Reply:
What do you mean by "all versions of LLVM 14" exactly ?

What I can already say is that llvm@15 works fine without the patch, as I have it in the deps of another env which works.

```
[+]  o2-aliceo2@dev%apple-clang@15.0.0+analysis~asan~ipo+sim~tsan~upgrades~usan build_system=cmake build_type=Release cxxstd=17 dev_path=/Users/laurent/alice/dev/O2 generator=ninja arch=darwin-ventura-m1
[+]      ^abseil-cpp@20230125.3%apple-clang@15.0.0~ipo+shared build_system=cmake build_type=Release cxxstd=17 generator=ninja arch=darwin-ventura-m1
[+]      ^arrow@11.0.0%apple-clang@15.0.0~brotli+compute~cuda+gandiva~glog~hdfs+ipc~ipo~jemalloc+lz4~orc~parquet~python+shared~snappy+tensorflow+zlib~zstd build_system=cmake build_type=Release generator=ninja arch=darwin-ventura-m1
[+]          ^flatbuffers@2.0.6%apple-clang@15.0.0~ipo~python+shared build_system=cmake build_type=Release generator=ninja arch=darwin-ventura-m1
[+]          ^llvm@15.0.7%apple-clang@15.0.0+clang~cuda~flang~gold~ipo~libomptarget~libomptarget_debug~link_llvm_dylib+lld~lldb+llvm_dylib~lua~mlir+polly~python~split_dwarf~z3~zstd build_system=cmake build_type=Release compiler-rt=runtime generator=ninja libcxx=none libunwind=runtime openmp=runtime patches=25bc503,8248141,90f3d78 shlib_symbol_version=none targets=all version_suffix=none arch=darwin-ventura-m1
...
```

Reply:
I mean LLVM 14.0.0 to 14.0.6